docs/configure.rst

```diff
@@ -0,0 +1,113 @@
+.. highlight:: console
+.. _configure:
+
+
+=========
+Configure
+=========
+
+
+Install Croud CLI
+=================
+
+This page shows you how to download and install the `Croud CLI`_. Croud is the dedicated command-line interface (CLI) tool to interact with CrateDB Cloud.
+The CLI provides specific commands to create and manage CrateDB Cloud
+organizations, projects, and resources. You can also use it to deploy CrateDB products such as clusters.
+
+In this tutorial, we refer to the *Croud CLI* as *Croud*.
+
+.. NOTE::
+
+   Croud requires `Python 3.6 or higher`_.
+
+.. SEEALSO::
+
+   Check out the `Croud CLI Reference`_ for an outline of all available Croud
+   commands, command-line options, and usage instructions.
+
+To install Croud, start your system's command-line application and run the
+following command:
+
+.. code-block:: console
+
+    sh$ python3 -m pip install --user -U croud
+
+Next, verify if the installation was successful by running:
+
+.. code-block:: console
+
+    sh$ croud -v
+
+If Croud was successfully installed, the command outputs the Croud version
+number, like so:
+
+.. code-block:: console
+
+   croud 0.16.0.
+
+
+.. _sign-in:
+
+Sign in to CrateDB Cloud
+========================
+
+This sections shows you how to sign in to CrateDB Cloud using `Croud`_.
```

Comment:
no need to into each section. the page intro should do the job of outlining the steps we're taking. each section can just get right to it